src/shared/styles/theme.ts

```diff
@@ -28,6 +28,10 @@ export const theme = createTheme({
           color: variables.palette.on_surface,
           height: '100vh',
           overflowY: 'hidden',
+          '.marked': {
```

Comment:
maybe rename for example to "highlighted-text" to make it more specific in case classes 'marked' will appear accidentally in the other place?